src/instructlab/model/evaluate.py

```diff
@@ -262,6 +262,7 @@ def launch_server(
 @click.option(
     "--benchmark",
     type=click.Choice([m.value for m in Benchmark.__members__.values()]),
+    required=True,
```

Comment:
Why is this flag required all of a sudden? The commit should probably explain why.

Reply:
Apologies, I had left my PR details inside of a comment block and didn't notice.  Fixed now but noting here the issue was `ilab model evaluate` returned without error/message.

Reply:
Does the code work with `benchmark = None`?

Reply:
> Does the code work with `benchmark = None`?

It does not.  Benchmark must be set or the command will fall through and do nothing.

Reply:
Yes, if we do not set a benchmark then `ilab model evaulate` does not know which evaluation scheme to use ergo is unable to import the corresponding evaluation class